requirements.txt

```diff
@@ -1,19 +1,19 @@
-h5py==3.1.0
-joblib==1.0.1
-markdown2==2.4.2
-numpy==1.21
-opencv-python-headless==4.5.1.48
-pip-licenses==3.3.1
-PTable==0.9.2
-PySide2==5.15.2
-PyYAML==6.0
-scikit-learn==0.24.1
-scipy==1.6.1
-Shapely==1.7.1
-shiboken2==5.15.2
-six==1.14.0
-sklearn==0.0
-tabulate==0.8.8
-threadpoolctl==2.1.0
-unittest-xml-reporting==3.0.2
-xgboost==1.2.1
```

Comment:
It's not provided with sklearn yet to my knowledge. I may end up doing a different branch that updates all the library requirements, because most of these numbers are from 3 years ago.

It shouldn't break anything here by removing it, just disabled the xgboost choice from classifiers.